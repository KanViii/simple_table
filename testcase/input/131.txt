INSERT a string
ASSIGN a 'd b c'