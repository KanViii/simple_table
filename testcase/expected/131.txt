Invalid: ASSIGN a 'd b c'